packages/mantine/src/components/prompt/Prompt.tsx

```diff
@@ -119,7 +121,7 @@ export const Prompt = factory<PromptFactory>((_props, ref) => {
                     <Modal.Body {...getStyles('body', stylesApiProps)}>
                         <Box {...getStyles('inner', stylesApiProps)}>{otherChildren}</Box>
                     </Modal.Body>
-                    {footer}
+                    {...footers}
```

Comment:
React handles rendering `ReactElement[]` on its own

```suggestion
                    {footers}
```